File: data/scripts/spritesheets/switch.py
import ugdk_math

def get_name(s):
    return "resources/images/switch/%04d.png" % s

file = [get_name(i+1) for i in range(120)]
frames = [[0, 0, 106, 106, ugdk_math.Vector2D(53, 93), i ] for i in range(120)]

File: data/scripts/spritesheets/switch.lua
require 'ugdk.math'

file = {}
frames = {}

for i = 1,120 do
  table.insert(file, "resources/images/switch/" .. string.format("%.4d", i) .. ".png")
  table.insert(frames, {0, 0, 106, 106, ugdk_math.Vector2D(53, 93), i - 1 })
end
